manifests/application.pp

```diff
@@ -15,6 +15,7 @@
   String $candlepin_oauth_key = $::katello::candlepin_oauth_key,
   String $candlepin_oauth_secret = $::katello::candlepin_oauth_secret,
   Stdlib::Httpsurl $pulp_url = $::katello::pulp_url,
+  Stdlib::Httpsurl $crane_url = $::katello::crane_url,
```

Comment:
The settings file generically referes to things as a regristry but this is specific to crane which is a bit of disconnect to me.